salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/pageObjects/account/accountImportPage/ClassicImportAccountPage.java

```diff
@@ -0,0 +1,63 @@
+package org.fundacionjala.salesforce.ui.pageObjects.account.accountImportPage;
+
+import org.fundacionjala.core.selenium.interaction.GuiInteractioner;
+import org.fundacionjala.salesforce.ui.pageObjects.account.accountImportPage.bulkDataLoadJobs.BulkDataLoadJobsPage;
+import org.fundacionjala.salesforce.ui.pageObjects.account.accountImportPage.editMapping.EditMappingPage;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * [SL] ClassicImportAccountPage.
+ */
+public class ClassicImportAccountPage extends AbstractImportAccountPage {
+
+    @FindBy(css = ".toolsContentRight div.lbBody ul li span [href*='Import']")
+    private WebElement importButton;
+
+    /**
+     * [SL] Wait the page load end.
+     */
+    @Override
+    protected void waitLoadPage() {
+        GuiInteractioner.waitLoadPage(importButton);
+    }
+
+    /**
+     * [SL] Clicks in the button or link for open the import account.
+     */
+    @Override
+    public void clickImportElement() {
+        GuiInteractioner.clickWebElement(importButton);
+    }
+
+    /**
+     * [SL] This method contain the first steps of the importation.
+     */
+    @Override
+    public void chooseDataStep(final String fieldEmail, final String fileName, final String option) {
```

Comment:
This method should return another diferent Step not the same; when you click "Next" button, where are you diven to?